manifests/params.pp

```diff
@@ -1,5 +1,14 @@
 class bootstrap::params {
-  $source_path     = '/usr/src/installer/'
-  $admin_user      = 'training'
-  $ec2_lock_passwd = true
+  $source_path      = '/usr/src/installer/'
+  $admin_user       = 'training'
+  $ec2_lock_passwd  = true
+
+
+  $pe_version       = '2017.1.0'
```

Comment:
I just picked & tried  some different numbers while I was testing the version checking logic. I don't actually imagine that it will be hard coded like this when it gets to production ready.